packages/storage-mixin/.depcheckrc

```diff
@@ -1,3 +1,3 @@
-ignores: [
-  "electron"
-]
+ignores:
+ - 'electron'
+ - '@electron/remote'
```

Comment:
Is there some issue with depcheck here? Doesn't look like we need to ignore this 🤔